Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -89,8 +89,10 @@ TerminalWidget::TerminalWidget(int ptm_fd, bool automatic_size_policy)
     : m_terminal(*this)
     , m_automatic_size_policy(automatic_size_policy)
 {
-    static_assert(sizeof(m_colors) == sizeof(xterm_colors));
-    memcpy(m_colors, xterm_colors, sizeof(m_colors));
+    VERIFY(m_colors.size() == xterm_colors.size());
```

Comment:
Also, doesn't static_assert still work here even with Array?

Reply:
No. I checked.

Reply:
```
/home/njs/Code/serenity/Userland/Libraries/LibVT/TerminalWidget.cpp: In constructor 'VT::TerminalWidget::TerminalWidget(int, bool)':
/home/njs/Code/serenity/Userland/Libraries/LibVT/TerminalWidget.cpp:92:35: error: non-constant condition for static assertion
   92 |     static_assert(m_colors.size() == xterm_colors.size());
      |
```

Reply:
Ah right, because `m_colors` is a member variable, and reading `this` in a constant expression won't work. Somewhat unfortunate, but ok.